applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -899,6 +913,19 @@ bool EmbeddedSkinVisualizationProcess::ElementIsSplit(
     return is_split;
 }
 
+bool EmbeddedSkinVisualizationProcess::ElementIsAusasIncised(const Vector &rEdgeDistancesExtrapolated)
+{
+    // Check whether one edge has intersection ratio with extrapolated skin if the vector is not empty
+    if (mShapeFunctionsType == ShapeFunctionsType::Ausas) {
```

Comment:
```suggestion
    KRATOS_ERROR_IF_NOT(mShapeFunctionsType == ShapeFunctionsType::Ausas) << "Trying to check if the element is incised but Ausas modified shape functions are not to be used." << std::endl;
```

Reply:
Would it still work then to have continuous shape funtions or dicontinuous shape functions without treating incised elements? Aren't they used anymore anyway?

Reply:
If the levelset is continuous it is impossible that incised elements appear as we always have a closed volume of fluid. About the Ausas comment, technically yes, the discontinuous levelset could be used with a different discontinuous FE, but as we have only the Ausas implemented (indeed I'm not aware of an alternative one) I'd throw the error to better inform the user about an eventual misbehavior.

Reply:
My point is, that this method is also called for continuous shape functions right now. So we either change that or it will always give an error for continuous shape functions, right?

Reply:
But this corresponds a bit to your comment above about calling IsSplit first

Reply:
It does not correspond.. so the thing is, I think, that we need to call `SetEdgeDistancesExtrapolatedVector` and `ElementIsIncised` also when continuous shape functions are used. It would return false for continuous shape functions anyway, because no `ExtrapolatedEdgeRatios` were calculated, right? But just to be sure, I added the check.

Reply:
On another node: The idea is that if discontinuous shape functions are used, then usually incised elements are considered by using extrapolated edge ratios and Ausas incised SF, right? So, maybe we should make the flag `CalculateElementalEdgeDistancesExtrapolated` true by default for the `CalculateDiscontinuousDistanceToSkinProcess`?

Reply:
OK. Now I see your point. Forget about the error thrown and keep the previous implementation, that is
- continuous case: always `false` inciseness
- discontinuous case: we check the inciseness

About activating the edge distances by default in the `CalculateDiscontinuousDistanceToSkinProcess` I think it is completely opposite. We want it to be deactivated by default as (so far) only our formulation supports it. Note that activating this by default would imply a useless overhead since the continuous case since this uses the discontinuous process under the hood.

Reply:
Ah right